src/Streaming/Cassava.hs

```diff
@@ -126,9 +126,12 @@ runParser :: (Monad m) => Parser a -> ByteString m r
              -> Stream (Of (Either CsvParseException a)) m (Either (CsvParseException, ByteString m r) r)
 runParser = loop
   where
-    feed f str = (uncurry (loop . f) . fromMaybe (mempty, str))
-                          -- nxt == Nothing, str is just Return
-                 =<< lift (B.unconsChunk str)
+    feed f str = do
+      nxt <- lift (B.nextChunk str)
+      let g = loop . f
+      case nxt of
+        Left r -> pure $ Right r
```

Comment:
Please align the `->`.

Alternatively, consider using the `either` function to avoid an explicit case statement.

Reply:
Aligned the `->` in both spots.